src/main/java/frc/robot/Constants.java

```diff
@@ -5,6 +5,49 @@
 package frc.robot;
 
 public final class Constants {
+  
+  //Shooter
+  public static final int SHOOTER_MOTOR_ID_1 = 44;
+  public static final int SHOOTER_MOTOR_ID_2 = 45;
+
+  public static final int SHOOTER_SENSOR_ID_1 = 0;
+  public static final int SHOOTER_SENSOR_ID_2 = 1;
+
+  public static final double SHOOTER_MOTOR_SPEED = 0.75;
+
+  public static final double SHOOTER_MOTOR_1_RPM = 12000;
+  public static final double SHOOTER_MOTOR_2_RPM = 12000;
+
+  //Shooter motor 1 PID constants
+  public static final double SHOOTER_MOTOR_1_PID_P = 5e-5;
+  public static final double SHOOTER_MOTOR_1_PID_I = 0;
+  public static final double SHOOTER_MOTOR_1_PID_D = 5e-5;
+  public static final double SHOOTER_MOTOR_1_PID_IZ = 0;
+  public static final double SHOOTER_MOTOR_1_PID_FF = 0.000015;
+  public static final double SHOOTER_MOTOR_1_MAX_OUTPUT = 1;
+  public static final double SHOOTER_MOTOR_1_MIN_OUTPUT = -1;
+  
+  //Shooter motor 2 PID constants
+  public static final double SHOOTER_MOTOR_2_PID_P = 5e-5;
+  public static final double SHOOTER_MOTOR_2_PID_I = 0;
+  public static final double SHOOTER_MOTOR_2_PID_D = 5e-5;
+  public static final double SHOOTER_MOTOR_2_PID_IZ = 0;
+  public static final double SHOOTER_MOTOR_2_PID_FF = 0.000015;
+  public static final double SHOOTER_MOTOR_2_MAX_OUTPUT = 1;
+  public static final double SHOOTER_MOTOR_2_MIN_OUTPUT = -1;
+
+
+  public static class OperatorConstants {
```

Comment:
This should not be in the OperatorConstants class, take a look how we created the 2 joysticks.